core/api/jrpc/value_converter.hpp

```diff
@@ -12,10 +12,37 @@
 #include "common/blob.hpp"
 #include "common/visitor.hpp"
 #include "primitives/extrinsic.hpp"
+#include "primitives/version.hpp"
 
 namespace kagome::api {
-  inline jsonrpc::Value makeValue(std::string v) {
-    return std::move(v);
+  inline jsonrpc::Value makeValue(common::Hash256 const &);
```

Comment:
Could you explain why did you put these declarations here?

Reply:
Because some of this methods calls overloaded ones. To make this code independent of functions definitions order we can use forward declaration.

Reply:
Ah, okay. You preferred to predeclare all of them instead of shuffling trying to put in a working order. Smart move.